src/event_bus.h

```diff
@@ -29,6 +29,7 @@ class event_bus
         void send( Args &&... args ) const {
             send( cata::event::make<Type>( std::forward<Args>( args )... ) );
         }
+        void send_dynamic( event_type type, std::vector<std::string> &args ) const;
```

Comment:
Rather than have `send_dynamic` here, I think it's better to have `event::make_dyn` take a runtime `event_type` and then people can build the even that way and pass it to the existing overload of `send` which takes an `event` object directly.